webpack.config.js

```diff
@@ -23,7 +25,88 @@ module.exports = {
           babelrc: false,
           presets: [['es2015', { modules: false }], 'stage-2']
         }
+      },
+      {
+        test: /\.(svg|png|jpg)$/,
+        loader: 'file-loader',
+        options: {
+          name: 'resources/[name].[hash].[ext]'
+        }
+      },
+      {
+        test: /\.css$/,
+        use: [
+          {
+            loader: 'file-loader',
+            options: {
+              name: 'resources/[name].[hash].[ext]'
+            }
+          },
+          'extract-loader',
+          'css-loader'
+        ]
+      },
+      {
+        test: /\.hbs$/,
+        use: [
+          {
+            loader: 'html-loader',
+            options: {
+              interpolate: 'require',
+              minimize: false
+            }
+          }
+        ]
       }
     ]
-  }
+  },
+  plugins: [
+    new CopyPlugin([
+      {
+        context: 'public',
+        from: 'locales/**/*.ftl'
+      },
+      {
+        context: 'public',
+        from: '*.*'
+      },
+      {
+        from: 'views/**',
+        to: '../'
+      },
+      {
+        context: 'node_modules/l20n/dist/web',
+        from: 'l20n.min.js'
+      },
+      {
+        context: 'node_modules/babel-polyfill/dist',
+        from: 'polyfill.min.js'
+      }
+    ]),
+    new HtmlPlugin({
+      filename: '../views/index.handlebars',
```

Comment:
Should we just rename all the `../views/*.handlebars` to `*.hbs`, for consistency?